fabric-command-api-v2/src/testmod/java/net/fabricmc/fabric/test/command/CommandTest.java

```diff
@@ -40,11 +41,11 @@
 
 	@Override
 	public void onInitialize() {
-		CommandRegistrationCallback.EVENT.register((dispatcher, dedicated) -> {
+		CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> {
 			// A command that exists on both types of servers
 			dispatcher.register(literal("fabric_common_test_command").executes(this::executeCommonCommand));
 
-			if (dedicated) {
+			if (environment == RegistrationEnvironment.DEDICATED) {
```

Comment:
Check `environment.dedicated` so that `RegistrationEnvironment.ALL` is handled properly.